libs/components/lookup/src/lib/modules/country-field/country-field.component.ts

```diff
@@ -57,6 +58,7 @@ let uniqueId = 0;
   styleUrls: ['./country-field.component.scss'],
   providers: [SKY_COUNTRY_FIELD_VALIDATOR],
   changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Can you respond to this thread with the reasoning for this change? Mainly for posterity's sake.

Reply:
This was required because `intl-tel-input` now sets values for the flag size on CSS custom properties on `:root` and country field was no longer able to see those values with view encapsulation turned on.